File: calibration/mutations/public-synthetic/fixtures/go-namesake/status.go
package status

type httpClient struct{}
func (httpClient) Get(string) {}

type database struct{}
func (database) Query(string) {}

type jwtToken struct{}
func (jwtToken) New() {}

func Ready() bool {
	httpClient{}.Get("local")
	database{}.Query("local")
	jwtToken{}.New()
	return true
}

File: calibration/mutations/public-synthetic/fixtures/kotlin-namesake/Status.kt
package sample

annotation class Query(val value: String)
annotation class PreAuthorize(val value: String)
annotation class Composable

class HttpClient {
    fun newCall(value: Any?) = value
}

data class Status(val ready: Boolean) {
    fun isReady(): Boolean = ready
}

@Query("local")
@PreAuthorize("local")
@Composable
fun localNames(client: HttpClient) {
    client.newCall(null)
    get("/local") { }
}

fun get(path: String, block: () -> Unit) = block()
